.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Create Snap Channel
+assignees: jardon
+labels: enhancement
+---
```

Comment:
This file shouldnt be needed for this as youre not using the action to create an issue.

Reply:
Yes. Definitely! Removed!